package.json

```diff
@@ -16,7 +16,7 @@
     "demo": "pnpm run --filter=demo.tutorialkit.dev dev",
     "demo:build": "pnpm run build && pnpm run --filter=demo.tutorialkit.dev build",
     "lint": "eslint \"{packages,docs,extensions,integration}/**/*\"",
-    "test": "pnpm run --stream --filter=@tutorialkit/* test --run"
+    "test": "CI=true pnpm run --stream --filter='@tutorialkit/*' test"
```

Comment:
Oh looks like this `CI=true` is a leftover from local testing.

Reply:
It was intentional here. Playwright won't run if `--run` is passed, and Vitest would start in watch-mode without it. But this change will be reverted once https://github.com/stackblitz/tutorialkit/pull/241#discussion_r1724721600 is done.